packages/blocks/src/attachment-block/attachment-block.ts

```diff
@@ -185,8 +185,14 @@ export class AttachmentBlockComponent extends CaptionedBlockComponent<
       })
     );
     // this is required to prevent iframe from capturing pointer events
-    this.handleEvent('pointerMove', ctx => {
-      this._isDragging = ctx.get('pointerState').dragging;
+    this.handleEvent('dragStart', () => {
+      this._isDragging = true;
```

Comment:
Why shouldn't we still use the `ctx.dragging`?

Reply:
In this PR implementation, `pointerMove` and `dragMove` events are managed independently to make the code easier to maintain. Therefore, the `pointerMove` event does not handle any drag-related logic, and neither of them shares the same `pointerEventState`.

In this settings, during the `dragMove` event, although the `pointerMove` event is also emitted, it does not know whether the current pointer is dragging or not. Thus, I have removed the `dragging` field from `pointerEventState`. Now, functions that need to detect dragging will use the `dragXXX` events.